frontend/src/data/config/MetricConfigSDOH.ts

```diff
@@ -7,18 +7,44 @@ import {
   populationPctShortLabel,
   populationPctTitle,
 } from './MetricConfigUtils'
+import { SHOW_GUN_VIOLENCE } from '../providers/GunViolenceProvider'
 
+console.log(SHOW_GUN_VIOLENCE)
```

Comment:
i actually like leaving this log in but it should say more, like "Gun violence preview enabled" or something